tomopy/misc/corr.py

```diff
@@ -188,6 +191,49 @@ def median_filter(arr, size=3, axis=0, ncore=None):
     e.shutdown()
     return out
 
+def median_filter_cuda(arr, size=3):
+    """
+    Apply median filter to 3D array along 0 axis with GPU support.
+    The winAllow is for A6000, for Tian X support 3 to 8
+    Parameters
+    ----------
+    arr : ndarray
+        Input array.
+    size : int, optional
+        The size of the filter.
+
+    Returns
+    -------
+    ndarray
+        Median filtered 3D array.
+    """
+
+    winAllow = [2,3,4,5,6,7,8,9.10,11,12,13,14,15]
```

Comment:
Hi tdmpelt:

the window size=1 and size>15 is not supported. Actually, GPU performs worse and worse when you increase the window size. 

Thanks